website/docs/r/group_profile.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "turbot"
+title: turbot
+template: Documentation
+page_title: "Turbot: turbot__group_profile"
+nav:
+  title: turbot__group_profile
+---
+
+# turbot_profile
+
+The `Turbot group profile` resource adds support for creating group profile under a directory type. It is used to create, manage and delete group profiles.
+
+## Example Usage
+
+**Creating Your First group profile**
+
+```hcl
+resource "turbot_group_profile" "test" {
```

Comment:
Please try to use examples that have more real life meaning. A good example / good name can be half the documentation without me reading any descriptions etc below.

when it's just "foo" or "test" or generic, it doesn't really teach me anything in reading the example.

e.g. compare to if this doc referred to an "Admins" group?